workspaces/liferay-sample-workspace/client-extensions/liferay-sample-data-set-filter/src/index.ts

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+
+// These definitions will ultimately be referenced by
+//
+// import type {FDSFilter} from '@liferay/js-api/data-set';
```

Comment:
You can add them if you want, since we haven't released any release version of `@liferay/js-api` yet. Just snapshots -> https://www.npmjs.com/package/@liferay/js-api?activeTab=versions